docs/user/03-60-formatting-service-binding-secret.md

```diff
@@ -0,0 +1,232 @@
+# Formats of Service Binding Secrets
```

Comment:
where do you make the split between instructional docs that are under "tutorials", and instructional docs that are under "user"? Meaning, while this doesn't have step instructions (though it could, with a bit of refactoring), doesn't this also answer a question of "how do I..." (in this case, ...specify the format of the secret)? 

If content could be both, descriptive (a DITA concept topic) or instructional (a DITA task topic), I'd argue it's always better to choose the instructional format, because that's more hands-on and thus more user-friendly.